### PR TITLE
Bump next from 12.0.7 to 14.1.0

### PR DESCRIPTION
**Pull request by basiq-ash**
Bumps [next](https://github.com/vercel/next.js) from 12.0.7 to 14.1.0.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v12.0.7...v14.1.0)

---
updated-dependencies:
- dependency-name: next dependency-type: direct:production ...

### CODE REVIEW

**Comment by basiq-ash** [comment]
@basiq-jon this update will fix all of the security vulnerabilities of this project. Tested on local and verified.

**Comment by basiq-ash** [comment]
Preview Test: https://account-verification-v3-ndypok0mc-basiq.vercel.app/

File: package.json [hunk 1]
```diff
@@ -5,30 +5,33 @@
...
     "next": "latest",
-    "qs": "^6.10.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-hot-toast": "^2.1.1",
+    "qs": "^6.11.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-hot-toast": "^2.4.0",
     "react-use-form-state": "^0.13.2"
```
Comment (basiq-jon) [inline package.json:21]: Can you just update this version rather than using a resolution to resolve the conflict?

**Review by basiq-jon — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (basiq-jon) [reply package.json:21]: Looks like there isn't a later version, it was last updated 4 years ago and is stuck on react 16 - probably should get rid of it at some point.

**Review by basiq-jon — APPROVED** [review]

**Pull request merged**